hack/verify-shellcheck.sh

```diff
@@ -30,9 +30,9 @@ DOCKER="${DOCKER:-docker}"
 
 # required version for this script, if not installed on the host we will
 # use the official docker image instead. keep this in sync with SHELLCHECK_IMAGE
-SHELLCHECK_VERSION="0.7.1"
-# upstream shellcheck latest stable image as of October 23rd, 2019
-SHELLCHECK_IMAGE="docker.io/koalaman/shellcheck-alpine:v0.7.1@sha256:d6147f30864ddb7c9cf983fc277d345bc315e798e309ddf70062b194843ee252"
+SHELLCHECK_VERSION="0.7.2"
+# upstream shellcheck latest stable image as of November 1st, 2022
```

Comment:
As you say in the description, latest stable is now 0.8.0.  I think this comment is unnecessary at this point
```suggestion
```

Reply:
You are absolutely right. Deleted that line.